pyro/distributions/torch/sparse_multivariate_normal.py

```diff
@@ -70,8 +70,9 @@ def log_prob(self, value):
     def _compute_logdet_and_mahalanobis(self, D, W, y, trace_term=0):
         """
         Calculates log determinant and (squared) Mahalanobis term of covariance
-        matrix (D + Wt.W), where D is a diagonal matrix, based on the
-        "Woodbury matrix identity" and "matrix determinant lemma":
+        matrix ``(D + Wt.W)``, where ``D`` is a diagonal matrix, based on the
+        "Woodbury matrix identity" and "matrix determinant lemma"::
```

Comment:
nit: itd be nice to put a link here to the [wiki](https://en.wikipedia.org/wiki/Woodbury_matrix_identity)

Reply:
meh it's a mere google search away